core/src/main/resources/hudson/tasks/Shell/config.groovy

```diff
@@ -27,3 +27,11 @@ f=namespace(lib.FormTagLib)
 f.entry(title:_("Command"),description:_("description",rootURL)) {
     f.textarea(name: "command", value: instance?.command, class: "fixed-width", 'codemirror-mode': 'shell', 'codemirror-config': "mode: 'text/x-sh'")
 }
+
+f.advanced() {
+
+    f.entry(title:_("Return code to set build unstable"), description:_("If set, the script return code that will be interpreted as an unstable build result.")) {
```

Comment:
Why is this inline, when the `BatchFile` equivalent is in the view's default locale resource file?